modules/pool/lib/apr-data-sources/ib-yield-apr-handlers/sources/default-apr-handler.ts

```diff
@@ -0,0 +1,183 @@
+import axios from 'axios';
+
+import { AprHandler } from '../ib-yield-apr-handlers';
+
+class DefaultAprHandler implements AprHandler {
+    tokens: string[];
+    url: string;
+    path: string;
+    scale: number;
+    networkPrismaId: string;
+    readonly group = 'DEFAULT';
+
+    constructor(aprHandlerConfig: DefaultAprHandlerConfig) {
+        this.tokens = aprHandlerConfig.tokens;
+        this.url = aprHandlerConfig.url;
+        this.networkPrismaId = aprHandlerConfig.network;
+        this.path = aprHandlerConfig.path ?? '';
+        this.scale = aprHandlerConfig.scale ?? 100;
+    }
+
+    async getAprs() {
+        try {
+            const { data } = await axios.get(this.url, { headers: { 'User-Agent': 'cf' } });
+            const value = this.path === '' ? data : this.getValueFromPath(data, this.path);
+            const scaledValue = parseFloat(value) / this.scale;
+
+            return this.tokens.reduce((acc, token) => {
+                acc[token] = scaledValue;
+                return acc;
+            }, {} as { [key: string]: number });
+        } catch (error) {
+            console.error(`Failed to fetch APRs in url ${this.url}:`, error);
+            return {};
+        }
+    }
+
+    getValueFromPath(obj: any, path: string) {
+        const parts = path.split('.');
+        let value = obj;
+        for (const part of parts) {
+            value = value[part];
+        }
+        return value;
+    }
+}
+
+export type DefaultAprHandlerConfig = {
+    tokens: string[];
+    url: string;
+    network: string;
+    scale?: number;
+    path?: string;
+};
+
+const vETHMainnet = '0x4bc3263eb5bb2ef7ad9ab6fb68be80e43b43801f';
```

Comment:
Like Aave, This could all be refactored into a map and all the handlers could be one function where you just pass in the network and it generates all handlers for that network on the fly and returns them in an array. Or you could pass in Network + Token and it returns a handler for that token if there is one defined.